controller/middleware/auth.go

```diff
@@ -0,0 +1,32 @@
+package middleware
+
+import (
+	"crypto/subtle"
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/labstack/echo"
+)
+
+var (
+	u = os.Getenv("BASIC_AUTH_USERNAME")
+	p = os.Getenv("BASIC_AUTH_PASSWORD")
+)
+
+// Init ...
+func Init() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
+}
+
+// BasicAuthUser ...
+func BasicAuthUser(username, password string, c echo.Context) (bool, error) {
```

Comment:
What is basic auth user do? please explain from the function and delete the comment

Reply:
BasicAuthUser is a basic function for Auth. It hasn't had any usage in the program yet.